src/components/Table/TableCell/TableCell.tsx

```diff
@@ -57,19 +62,33 @@ const TableCell: FC<TableCellProps> = ({
       [styles.compact]: isCompact,
       [styles.borderless]: isBorderless,
       [styles.truncated]: truncateOverflow,
+      [styles.sticky]: isColumnSticky,
       [styles['align-right']]: align === 'right',
       [styles['align-center']]: align === 'center',
     },
     className,
   );
 
   return (
-    <td
-      className={tableCellClasses}
-      style={{ ...width && { minWidth: `${width}px`, maxWidth: `${width}px` } }}
-    >
-      {(children === null || typeof children === 'undefined' || children === '') ? emptyCellPlaceholder : children}
-    </td>
+    <>
+      {isColumnSticky
+        ? (
+          <th
```

Comment:
since a `th` can be used for a column or row, we should add the `scope` property. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th

Reply:
Good catch! Will update this evening and add some tests tomorrow a.m.